lib/applications/resolvers.ts

```diff
@@ -0,0 +1,61 @@
+import { ApolloError } from 'apollo-server-errors'; // Apollo error
+import { Resolver } from '@lib/resolvers'; // Resolver type
+import {
+  ShopifyConfirmationNumberAlreadyExistsError,
+  ApplicantIdDoesNotExistError,
+  ApplicationFieldTooLongError,
+} from '@lib/applications/errors'; // Employee errors
+import { DBErrorCode } from '@lib/db/errors'; // Database errors
+
+/**
+ * Query all the RCD applications in the internal-facing app
+ * @returns All RCD applications
+ */
+export const applications: Resolver = async (_parent, _args, { prisma }) => {
+  const applications = await prisma.application.findMany();
+  return applications;
+};
+
+/**
+ * Create an RCD application
+ * @returns Status of operation (ok, error)
+ */
+export const createApplication: Resolver = async (_, args, { prisma }) => {
+  const {
+    input: { applicantId, shopifyConfirmationNumber },
+  } = args;
+
+  let application;
+  try {
+    application = await prisma.application.create({
+      data: { ...args.input },
+    });
+  } catch (err) {
+    if (
+      err.code === DBErrorCode.UniqueConstraintFailed &&
+      err.meta?.target.includes('shopifyConfirmationNumber')
+    ) {
+      throw new ShopifyConfirmationNumberAlreadyExistsError(
+        `Application with Shopify confirmation number ${shopifyConfirmationNumber} already exists`
+      );
+    } else if (
+      err.code === DBErrorCode.ForeignKeyConstraintFailed &&
+      err.meta.target.includes('applicantId')
```

Comment:
```suggestion
      err.meta?.target.includes('applicantId')
```

if `err.meta` can be undefined, it should be this.